extensions/index.md

```diff
@@ -2,22 +2,101 @@
 layout: page
 title: Extensions
 show_sidebar: true
+redirect_from: /profiles
 ---
 
-## <a href="#status" id="status" class="headerlink"></a> Status
+JSON:API can be extended with profiles. These profiles enable an API to 
+provide clients with information or functionality beyond that described 
+in the base JSON API specification.
 
-**An extension system is currently under development,** and you can view the
-latest work [here](https://github.com/json-api/json-api/tree/profile-extensions).
-There is no official support for extensions in the base JSON API specification.
+Anyone can author a profile, and a single profile can be reused by multiple APIs.
+Popular profiles may be implemented by off-the-shelf tools so that developers 
+can seamlessly take advantage of the features these profiles provide.
+
+## <a href="#existing-profiles" id="existing-profiles" class="headerlink"></a> Existing Profiles
+
+{% for category in site.profile_categories %}
+  <h3 id="#profiles-category-{{ category | slugify }}">
+    {{ category }}
+  </h3>
+  <dl class="profiles-list">
+    {% for profile in site.profiles %}
+      {% if profile.categories contains category %}
+        <dt><a href="{% include profile_url.md page=profile %}">{{ profile.name }}</a></dt>
+        <dd>{{ profile.short_description }}</dd>
+      {% endif %}
+    {% endfor %}
+  </dl>
+{% endfor %}
+
+## <a href="#profile-creation" id="profile-creation" class="headerlink"></a> Creating a New Profile
+
+### <a href="#profile-creation-before" id="profile-creation-before" class="headerlink"></a> Before Creating a Profile
+
+Please **check whether an existing profile fits your needs or could be amended
+to fit your needs** before developing a new one.
+
+- If a suitable profile already exists, consider using it. Having fewer, more
+widely-deployed profiles makes it easier to create shared tooling.
+
+- If there's an existing profile that could be amented to fit your needs,
+consider asking the profile's author if they would be willing to modify it as
+needed. Contact them through the information in their profile's registration
+and give them some time to reply.
+
+### <a href="#profile-creation" id="profile-creation" class="headerlink"></a> Authoring Your Profile
+
+To author your profile, [download and fill out the template](/profile_template.md).
+
+### <a href="#profile-register" id="profile-register" class="headerlink"></a> Register & Use Your Profile
+
+Once it's ready, submit your profile to the JSON:API profile registry by creating
+a pull request that contains your filled out template as an `index.md` file in a 
+uniquely-named folder in the [`_profiles`](https://github.com/json-api/json-api/tree/gh-pages/_profiles) 
+directory.
+   
+Your submission will be **approved within a week** by one of JSON API's 
```

Comment:
Really? I think that's an ambitious promise! Are you sure?

Reply:
Fair enough!

Reply:
Personally, I'm unable to make any promise to review profiles within a short timeframe, and I'm not sure it's desirable to force this either. If we or the community see an obvious design flaw in a profile, isn't it preferable to point out the flaw rather than to accept it? (But then again, I'm not comfortable allowing too much extra noise into this issue tracker by using it for extensive reviews.)

All of this to say that I feel that we shouldn't be guaranteeing "approval within a week" let alone "review within a week". I'm more comfortable saying something along the lines of "Your submission will be reviewed in a timely manner by JSON:API's editors and the community."

Reply:
Yes, if we see a glaring flaw in a profile's design, I definitely think we should point it out. And I imagine that, if we point out a major flaw, the profile's author will generally address it before registering the final version. 

I can clarify the text in this PR to say that, if the community gives feedback, the author can of course take time to act on that feedback, which then would prompt a new round of review, and that this whole process could take more than a week.

I'm really worried, though, about us coming to see the approval process as a "design review", because design reviews tend to take a lot of time/attention, and can get nitpicky. That's frustrating to authors and is likely not a good use of our very-limited bandwidth as editors.

Because profiles are decentralized, I don't see a lot of harm if a poorly-designed profile get registered; people can just not use that profile. It doesn't bind new APIs or hold back the spec. The second review criterion I proposed above ("if this profile did become widely adopted, would that cause any sort of problems?"), which I'm also happy to make more explicit in the PR's text, is meant to ensure that a registered profile wouldn't hold back the ecosystem either.

Given the limited harm, I'd much rather have some bad profiles get through, then have some basically well-designed profiles languish for weeks because we don't have time to review their designs. At least at first, people will likely be developing profiles for their project's immediate use case, and I don't think we can ask them to put their project on hold for more than a week while we're reviewing/asking for changes. 

If the issue is how to help implementers avoid using bad profiles, perhaps we can do something like link to the each profile's review thread next to any links to the its spec. Or, we can exercise our editorial discretion to highlight profiles we think our well designed by listing them above others. Whatever the approach, I don't think this is a hard problem to solve.

All told, I feel very strongly about the importance of quick review, and I'm happy to take on the task of reviewing all incoming profiles according to the criteria I gave above. If the one week timeline proves impossible, we can always change things then. 

Again, I think a critical value of profiles is their potential to unblock the community and get people sharing solutions — which I think means showing the profiles on jsonapi.org — and the last thing I want is for review to become a new bottleneck.

Reply:
See updated test.